sklearn/model_selection/tests/test_search.py

```diff
@@ -165,6 +165,24 @@ def test_grid_search():
     assert_raises(ValueError, grid_search.fit, X, y)
 
 
+def test_grid_search_incorrect_param_grid():
+    assert_raise_message(
+        ValueError,
+        "Parameter values for parameter (C) need to be a sequence.",
+        GridSearchCV,
```

Comment:
We would not usually space things out this much. Put `GridSearchCV` and its parameters on the same line, I think.